src/api/app/services/scm_status_reporter.rb

```diff
@@ -7,51 +7,10 @@ def initialize(event_payload, event_subscription_payload, scm_token, event_type
     @state = event_type.nil? ? 'pending' : scm_final_state(event_type)
   end
 
-  # rubocop:disable Metrics/CyclomaticComplexity
-  # rubocop:disable Metrics/PerceivedComplexity
   def call
-    if github?
```

Comment:
With the removal of this line, we can also delete the `github?` method since it's not needed anymore.